.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space before `db.sqlite3`. It's a good practice to remove any leading or trailing spaces to ensure the `.gitignore` works as expected.